libpod/top.go

```diff
@@ -0,0 +1,37 @@
+package libpod
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/utils"
+	"github.com/sirupsen/logrus"
+)
+
+// This can be removed is stats is merged
+var CGroupParent = "/libpod_parent"
+
+// GetContainerPids reads sysfs to obtain the pids associated with the container's cgroup
```

Comment:
Should this go in stats somewhere? I feel like either stats or inspect should be returning this. We can keep this too, I just think it should also come out elsewhere.

Reply:
Inspect and stats dont deal with specific Pids, hence the difference. It could land in container.go too for that matter.